src/lib/index.ts

```diff
@@ -80,6 +80,10 @@ export interface MoveOpts extends PathFinderOpts {
    * return `undefined`.
    */
   routeCallback?: (roomName: string, fromRoomName: string) => number | undefined;
+  /**
+   * Creep used capacity and body, used to calculate default terrain costs.
+   */
+  creepMovementInfo?: { usedCapacity: number; body: Creep['body'] };
```

Comment:
Let's prefix this with an underscore and add a note that it's for internal use only and should not be set manually. Probably want to refactor this later, but for now I think this will be good!